### DISCUSSION TITLE
Import avi

### DISCUSSION THREAD

**Discussion by Tailchasing** [fastplotlib/fastplotlib#329] [Q&A] [answered]
Hello! What will you recommend for importing .avi files followed by fastplotlib analysis? I am trying to establish an analysis pipeline for many .avi files (calcium imaging). My plan is to use the last frame for cell segmentation. I know this is a super naive question, and I would really appreciate any help for a Python beginner. THANK YOU!

**Comment by kushalkolar** [comment, marked as answer]
I would recommend mesmerize-core which was made for this purpose:
https://github.com/nel-lab/mesmerize-core

It provides an API for lazy loading large video files (LazyVideo).

If you have issues feel free to post on the mesmerize-core repo :)

**Reply by Tailchasing** [reply]
Thank you! I will try!
